includes/controller/users_controller.php

```diff
@@ -433,6 +433,11 @@ function shiftCalendarRendererByShiftFilter(ShiftsFilter $shiftsFilter)
         return new ShiftCalendarRenderer($shifts, $needed_angeltypes, $shift_entries, $shiftsFilter);
     }
 
+    $own_shift_ids = [];
+    if (in_array(ShiftsFilter::FILLED_OWN, $shiftsFilter->getFilled())) {
+        $own_shift_ids = Shifts_by_user(auth()->user()->id, false)->pluck('id')->toArray();
```

Comment:
```suggestion
        $own_shift_ids = Shifts_by_user(auth()->user()->id)->pluck('id')->toArray();
```
false is the default so it's not needed